<launch>
    <node name="filter" pkg="filter" type="filter_node" output="screen">
        <param name="map" value="/map" />
        <param name="frontier" value="/frontier_marker" />
        <param name="aim" value="/filtered_points" />
        <param name="radius" value="2" />
        <param name="wall_threshold" value="100" />
        <param name="base_frame" value="base_footprint" />
    </node>
</launch>
